app/views/hyku/api/v1/user/_user.json.jbuilder

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+json.id user.id
+json.email user.email
+json.created_at user.created_at
+json.updated_at user.updated_at
+json.guest user.guest
+json.facebook_handle user.facebook_handle
+json.twitter_handle user.twitter_handle
+json.googleplus_handle user.googleplus_handle
+json.display_name user.display_name
+json.address user.address
+json.admin_area user.admin_area
+json.department user.department
+json.title user.title
+json.office user.office
+json.chat_id user.chat_id
+json.title user.title
+json.office user.office
+json.chat_id user.chat_id
+json.website user.website
+json.affiliation user.affiliation
+json.telephone user.telephone
+json.avatar_file_name user.avatar_file_name
+json.avatar_content_type user.avatar_content_type
+json.avatar_file_size user.avatar_file_size
+json.avatar_updated_at user.avatar_updated_at
+json.group_list user.group_list
+json.groups_last_update user.groups_last_update
+json.linkedin_handle user.linkedin_handle
+json.orcid user.orcid
+json.arkivo_token user.arkivo_token
+json.arkivo_subscription user.arkivo_subscription
+json.zotero_token user.zotero_token
+json.zotero_userid user.zotero_userid
+json.preferred_locale user.preferred_locale
```

Comment:
The names of the json keys seems to match exactly with the field names in the User table, then perhaps you can consider making it shorter with:

````
json.merge! user.attributes.slice(:id, :email, :created_at, :updated_at, :guest, :facebook_handle, :twitter_handle, :googleplus_handle, :display_name, :address, :admin_area, :department, :title, :office, :chat_id, :website, :affiliation, :telephone, :avatar_file_name, : avatar_content_type, :avatar_file_size, :avatar_updated_at,  :group_list, :groups_last_update , :linkedin_handle, :orcid, :arkivo_token, :arkivo_subscription, :zotero_token, :zotero_userid,  :preferred_locale)
````

Reply:
I think I'd like to keep it consistent with the other endpoints but would be happy to take some more input